src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.ts

```diff
@@ -0,0 +1,69 @@
+import { UserService } from 'ngx-login-client';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { Observable, ConnectableObservable } from 'rxjs';
+
+import { Broadcaster } from 'ngx-base';
+import { Contexts } from 'ngx-fabric8-wit';
+import { WorkItemService, WorkItem } from 'fabric8-planner';
+
+import { DummyService } from './../shared/dummy.service';
+
+class WorkItemFilter {
+
+  paramKey: string;
+  value: string;
+  active: boolean;
+
+}
+
+@Component({
+  selector: 'fabric8-create-work-item-widget',
+  templateUrl: './create-work-item-widget.component.html',
+  styleUrls: ['./create-work-item-widget.component.scss']
+})
+export class CreateWorkItemWidgetComponent implements OnInit {
+
+  private _myWorkItems: ConnectableObservable<WorkItem[]>;
+  myWorkItemsCount: Observable<number>;
+  contextPath: Observable<string>;
+
+  constructor(
+    private context: Contexts,
+    private broadcaster: Broadcaster,
+    private workItemService: WorkItemService,
+    private userService: UserService,
+    private contexts: Contexts
+  ) { }
+
+  ngOnInit() {
+    this.contextPath = this.contexts.current.map(context => context.path);
+    this._myWorkItems = this.userService
+      .getAllUsers()
```

Comment:
Yes, but we're still waiting on the replacement (get all collaborators on a space).